addons/event_sale/models/event_registration.py

```diff
@@ -60,6 +46,11 @@ def _compute_utm_medium_id(self):
             elif not registration.utm_medium_id:
                 registration.utm_medium_id = False
 
+    def _is_free(self):
+        so = self.sale_order_id
+        so_line = self.sale_order_line_id
+        return super()._is_free and (not so or float_is_zero(so_line.price_total, precision_digits=so.currency_id.rounding))
```

Comment:
```suggestion
        return super()._is_free() and (not so or float_is_zero(so_line.price_total, precision_digits=so.currency_id.rounding))
```

But this method seems unused. Make sure to use it in the intended place.